lib/authorize_net/api/transaction.rb

```diff
@@ -257,5 +257,9 @@ def get_transaction_list_for_customer(request)
     def update_merchant_details(request)
       make_request(request, UpdateMerchantDetailsResponse, Type::API_UPDATE_MERCHANT_DETAILS)
     end
+
+    def get_customer_payment_profile_nonce(request)
```

Comment:
Please remove this function. We had planned not to make this API public.

Reply:
I have removed get_customer_payment_profile_nonce(request) function from transaction.rb and committed the changes.